model/generate.go

```diff
@@ -443,24 +443,35 @@ func (g *GeneratedProject) filterInactiveTasks(tasks TVPairSet, v *Version, p *P
 }
 
 type specificActivationInfo struct {
-	stepbackTasks      map[string][]specificStepbackInfo // tasks by variant that are being stepped back, along with the stepback depth to use
-	activationTasks    map[string][]string               // tasks by variant that have batchtime or activate specified
-	activationVariants []string                          // variants that have batchtime or activate specified
+	stepbackTasks            map[string][]specificStepbackInfo // tasks by variant that are being stepped back, along with the stepback depth to use
+	activationTasks          map[string][]string               // tasks by variant that have batchtime or activate specified
+	activationVariants       []string                          // variants that have batchtime or activate specified
+	generatedProjectVariants []parserBV                        // list of all variants that are specified in the generated project
```

Comment:
If this is only used by the dependency includer can it be added to that instead (same as deactivateGeneratedVariantTasks)?

Reply:
I've since moved this to be an additional [parameter](https://github.com/evergreen-ci/evergreen/pull/6111#discussion_r1042650009) to `IncludeDependencies`